src/org/jitsi/meet/test/util/TestUtils.java

```diff
@@ -305,6 +305,25 @@ public Boolean apply(WebDriver d)
                 });
     }
 
+    public static void waitForElementContainsClassByXPath(
```

Comment:
LGTM, we just need some javadoc above. This isn't a blocker for merging the commit, but please add it in a separate PR rather sooner than later, so that we don't forget.